controls/V-73241.rb

```diff
@@ -28,5 +31,9 @@
     describe registry_key('HKLM\SOFTWARE\McAfee\Endpoint\AV\ProductVersion') do
       it { should exist }
     end
+    describe "Windows Defender" do
+      subject { windefend }
+      it { should eq "Running" }
+    end
```

Comment:
This test was added in response to #3 in order to check if Windows Defender was enabled.